packages/components/src/components/assets/AssetIcon/AssetIcon.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from 'react';
+import {
+    AssetShareIndicator,
+    AssetShareIndicatorProps,
+} from '../AssetShareIndicator/AssetShareIndicator';
+import { useTheme } from 'styled-components';
+
+export const UPDATED_ICONS_LIST_URL_BASE = 'https://data.trezor.io/suite/icons/coins/';
+
+const QUALITY = {
+    small: '',
+    big: '@2x',
+};
+
+const QUALITY_SIZE = {
+    small: 24,
+    big: 48,
```

Comment:
nitpick: Are we going to add also normal in between in the future? Otherwise I would also rename it to `normal`

Reply:
I don't know, this is just a first idea. We could also totally avoid using small/big/normal namings here and implement the expected differently.

Reply:
I think small/normal/big makes sense, it's kind of predictable when we try to use it everywhere